vqpy/__init__.py

```diff
@@ -41,6 +42,9 @@ def launch(cls_name,
         detector_model_dir: the directory for all pretrained detectors.
         detector_name: the specific detector name you desire to use.
     """
+    # explicitly reset Tracker each time we start a new run
+    ByteTracker.Data.reset()
+
```

Comment:
I guess what you mean is put `reset()` in `MultiTracker.__init__`:
`setup_ground_tracker` will be invoked for each VObj type while we wish to reset only when we start tracking a new video.

Reply:
Putting it in `MultiTracker.__init__` does make sense, but it could be possible that multiple `MultiTracker`'s are created in the same video.
Plus this is a work-around, probably make it more explicit than hiding it in Tracker's inheritance structure.

Reply:
@whzzt 
Does that make sense, the `byte_trackers` in `multiTracker` share states? 
Or do we need a brand new byte_tracker for each vobj class?

Reply:
@hanryxu 
We shouldn't allow multiple applications to run in one process. Each e2e test should run in one process to avoid interference.

Reply:
Maybe as a workaround, we can add teardown fixtures to pytest that resets `ByteTrack` for now?

Reply:
Maybe wrap it in multi-tracker, and call `tracker.clear()` at the end of `__init__`. `ByteTracker` is not at the abstraction level of the main running process. The main running process includes the multi-class tracker, and the multi-class tracker includes single-class tracker, where `ByteTracker` is one single-class tracker.